corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -263,7 +263,7 @@
                 <div class="accordion-body collapse 
                     {% if cases|length == 1 %}in{% endif %}" 
                     id="form-case-acc-{{ forloop.counter }}">
-                    <div class="accordion-inner">
+                    <div class="accordion-inner" style="display: inline-block">
```

Comment:
@orangejenny, would you add an id to this node and target that in the stylesheet over using an in-line style?